akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -241,7 +255,13 @@ final class ORSet[A] private[akka] (
   private[akka] def add(node: UniqueAddress, element: A): ORSet[A] = {
     val newVvector = vvector + node
     val newDot = VersionVector(node, newVvector.versionAt(node))
-    assignAncestor(new ORSet(elementsMap = elementsMap.updated(element, newDot), vvector = newVvector))
+    val newDelta = _delta match {
+      case None ⇒ new ORSet(Map(element → newDot), newDot, deltaOp = AddDeltaOp)
+      case Some(d) ⇒
+        // FIXME handle merge/accumulation of deltas
```

Comment:
perhaps we should not support deltas for mixed add and deletes within the same Update, we can fall back to full state for such rare situations